pstress/pstress-run.conf

```diff
@@ -139,27 +139,43 @@ QUERIES_PER_THREAD=100000
 MYINIT=""
 
 ################################################################################
-# Extra options to pass to mysqld. Examples below                              #
+# Extra options to pass to mysqld.                                             #
 ################################################################################
 MYEXTRA=""
 
 ################################################################################
 # Keyring options: Component and Plugin                                        #
 # Both are supported in MySQL from 8.0.24 onwards. However, we can use either  #
-# component or plugin at a time. By default component will be used with PS-8.0 #
-# For PS-5.7, keyring plugin will be used                                      #
+# component or plugin at a time.                                               #
+# For PS-8.0, keyring_component will be used (default behaviour)               #
+# For PS-5.7, keyring_file plugin will be used                                 #
```

Comment:
Keyring component was introduced in version 8.0, but it is going to be there in future versions of MySQL as well eg 9.0, so I think we should not tie this feature with a specfic PS version.

Secondly, keyring component should also work with PXC80. Currently, it does not work and is reported as a bug. Once it will be fixed we will need to have this variable in pstress-run-PXC80.conf as well.